src/drivers/uart/src/i8250.rs

```diff
@@ -0,0 +1,138 @@
+// http://www.ti.com/lit/ds/symlink/pc16550d.pdf
+//use core::ops;
```

Comment:
In general, this file has way more commented-out code than is usually justified. Most code standards don't want to see any. Is all of it valid and interesting?

Reply:
for an early stage project like this, I'm ok with it. It's a sort of TODO pointer. Later on, standards will have to be much stricter.